docs/developers/protocol/hermez-protocol/protocol.md

```diff
@@ -217,15 +217,15 @@ The exit tree has the same leaf structure as the state tree with some particular
 - nonce is always set to 0
 - if several exits are done in the same batch for the same account, the balance is just added on top of the account
 
-User will need to prove that it owns a leaf in the exit tree in order to perform its withdraw and get back the tokens from the contract. This verification could be done either by submitting a merkle tree proof or by submitting a zkProof.
+User will need to prove that it owns a leaf in the exit tree in order to perform its withdraw and get back the tokens from the contract. This verification could be done either by submitting a Merkle tree proof or by submitting a zkProof.
 
 ## Account types
 
 ### Regular rollup account
 
-Regular accounts contain an ethereum address and a babyjubjub public key.  Accounts are always indexed by ethereum address in the UX, so it is a requirement that the ethereum address authorizes the account keys.  Once the account is created, the ethereum key is used to authorize L1 txs and the babyjubjub key is used to authorize L2 txs.
-There are two ways to authorize an account creation (that is, an ethereum address authorizes the creation of an account containing that same ethereum address and a babyjubjub public key):
-- Via ethereum transaction, which has an implicit signature of the ethereum address.  This requires the owner of the ethereum address to sign the smart contract transaction call
+Regular accounts contain an Ethereum address and a babyjubjub public key.  Accounts are always indexed by Ethereum address in the UX, so it is a requirement that the ethereum address authorizes the account keys.  Once the account is created, the ethereum key is used to authorize L1 txs and the babyjubjub key is used to authorize L2 txs.
```

Comment:
ethereum address authorizes => Ethereum address authorizes

the ethereum key is used to authoriz => the Ethereum key is used to....